content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,112 @@
-============================
-Reports for a Better Support
-============================
+=========
+Reporting
+=========
 
-An efficient customer service solution should have a built-in reporting option. Reports allow you to
-track trends, identify areas for improvement, manage employees’ workloads and, most importantly,
-meet your customer’s expectations.
+Odoo :guilabel:`Helpdesk` features several reports that provide the opportunity to track trends,
+identify areas for improvement, manage employees` workloads, and confirm when customer expectations
+are met.
 
-Cases
-=====
 
-Some examples of the reports Odoo Helpdesk can generate include:
+Available reports
+=================
 
-- The number of tickets *grouped by* team and ticket type.
+The reports below can be found under :menuselection:`Helpdesk --> Reporting`.
 
-| In this manner, you are able to evaluate which ticket types have been the most frequent ones,
-  plus the workload of your teams.
-| Apply *Time Ranges* if you would like to make comparisons to a *Previous Period* or a *Previous
-  Year*.
+Tickets Analysis
+----------------
 
-.. image:: reports/report_team_ticket_type.png
+The :guilabel:`Tickets Analysis` report provides an overview of every ticket in the database. The
+default report counts the number of tickets per team and groups them by stage.
+
+.. image:: reports/reports-tickets-default.png
    :align: center
-   :alt: View of a helpdesk ticket analysis by team and ticket type in Odoo Helpdesk
+   :alt: View of Tickets Analysis report default view
 
-- The number of tickets closed per day, per team.
 
-Get an overview of how many requests each team is closing per day in order to measure their
-performance. Identify productivity levels to understand how many requests they are able to handle.
+Alternative measures can be selected to track where the most time is spent at different points in
+the workflow. To change the metric used in the current report, choose an option from the
+:guilabel:`Measures` dropdown.
```

Comment:
Same as below - is it worth mentioning some of the different Measures available?